### PR TITLE
style(menuSelect): apply select theme

### LINKED ISSUE #2448: Style / menuselect: should have the same look as the other select with the theme
The widget is not styled for the moment.

screen shot 2017-10-04 at 12 23 25

It should have this style:

screen shot 2017-10-04 at 12 23 48

### PR DESCRIPTION
**Pull request by iam4x**
fixes #2448

### CODE REVIEW

File: src/css/theme/_hits-per-page.scss [hunk 1]
```diff
@@ -1,5 +1,8 @@
 select.ais-hits-per-page-selector {
+  -webkit-appearance: none;
```
Comment (bobylito) [inline src/css/theme/_hits-per-page.scss]: Don't we use autoprefixer?

Comment (bobylito) [reply src/css/theme/_hits-per-page.scss]: humm only on docgen...

**Comment by iam4x** [comment]
We do have autoprefixer, but this is an edge case where Chrome understands it without the prefix but does not applies.

So we can target lower browsers, but it would mean that other properties which we don't want to prefix would be prefixed it we fix it at postcss.config.js level 👍

**Comment by bobylito** [comment]
OK sounds fair @iam4x

**Pull request merged**